astroid/__init__.py

```diff
@@ -107,8 +107,22 @@ def inference_tip(infer_function):
 
        MANAGER.register_transform(Call, inference_tip(infer_named_tuple),
                                   predicate)
+
+    .. Note::
+
+        ALWAYS use a predicate with an inference_tip
+        since registering a transform will
+        influence inference: an interance tip
+        overwrites the _explicit_inference
```

Comment:
I find this a bit too technical, let's just say that currently when we define an inference tip, it will override any previously set inference tip for the same kind of node, leading to some of them not being called.